Gopkg.toml

```diff
@@ -41,31 +41,31 @@
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.16.2"
+  version = "kubernetes-1.17.0-rc.2"
```

Comment:
We shouldn't use the rc version.

Reply:
I am in the process of pushing a new commit with `v0.17.0` tag. Will push the changes once I done.

Reply:
Thanks for reviewing

Reply:
Thanks for reviewing